cwatm/hydrological_modules/water_demand/water_demand.py

```diff
@@ -1757,17 +2196,21 @@ def dynamic(self):
             if self.var.modflow:
                 if self.var.GW_pumping:  # pumping demand is sent to ModFlow (used in transient module)
                     # modfPumpingM is initialized every "modflow_timestep" in "groundwater_modflow/transient.py"
+                    self.var.modfPumpingM = globals.inZero.copy()
```

Comment:
To look into

Reply:
What would be the problem with it? Is it because it is initiated in the transient.py?
I can remove it

Reply:
Is there a reason it was included here?